relay-general/src/store/transactions/processor.rs

```diff
@@ -13,10 +13,6 @@ use crate::types::{Annotated, Meta, ProcessingAction, ProcessingResult, Remark,
 /// Configuration around removing high-cardinality parts of URL transactions.
 #[derive(Clone, Debug, Default)]
 pub struct TransactionNameConfig<'r> {
-    /// True if transaction names scrubbed by regex patterns should be marked as [`TransactionSource::Sanitized`].
-    ///
-    /// Transaction names modified by clusterer rules are always marked as such.
-    pub mark_scrubbed_as_sanitized: bool,
     /// Rules for identifier replacement that were discovered by Sentry's transaction clusterer.
     pub rules: &'r [TransactionNameRule],
```

Comment:
Even if there's only `rules` left in the config, I'm planning to leave it as-is for now, and remove the config struct in a follow-up next week.